<launch>
	<group ns="vision">
		<node pkg="stereo_od" type="od_node" name="stereo_od">
			<remap from="points"      to="narrow/points2"/>
			<remap from="camera_info" to="narrow/left/camera_info"/>
			<param name="points_min"   value="100"/>
			<param name="height_min"   value="0.5"/>
			<param name="height_max"   value="1.0"/>
			<param name="distance_max" value="7.0"/>
			<param name="theta"        value="1.04719755"/>
		</node>
	</group>
</launch>
